generative_LSTM_load.yml

```diff
@@ -0,0 +1,18 @@
+definitions:
+  ml_methods:
+    G1:
+      LSTM:
+        cores_for_training: 2
+  reports:
+    GeneratorReport: GeneratorReportLSTM
+instructions:
+  machine_learning_instruction:
+    type: GenerativeModelLoad
```

Comment:
Maybe there is a better name for the instruction? Is it just loading the model or is the aim to generate sequences from that model? :)

Reply:
The aim is to load a model, then generate using the pretrained model